README.md

```diff
@@ -59,60 +72,55 @@ You'll need this package structure to construct a provider package repo:
 ```
 
 
-## Development Standards
+## Part 2: Development Standards
 
-### Building Provider Package
+If you followed the formatting guidelines above, you're now ready to start editing files to include standard package functionality.
 
-Most of what you need is included in `setup.py` and ready to customize. You may use `setup.cfg` as a valid alternative.
+### Python Packaging Scripts
 
-### Provider Readmes
+Your `setup.py` file should contain all of the appropriate metadata and dependencies required to build your package. Use the [sample `setup.py` file](https://github.com/astronomer/airflow-provider-sample/blob/main/setup.py) in this repository as a starting point for your own project.
```

Comment:
Why is this `setup.py` file not within the provider package folder itself? We should keep all "example" files contained in a directory within the repo, and separate this README from that simulation.